hw/ip/rv_dm/rv_dm.core

```diff
@@ -13,6 +13,7 @@ filesets:
       - lowrisc:tlul:adapter_host
       - pulp-platform:riscv-dbg:0.1
     files:
+      - rtl/rv_dm_pkg.sv
```

Comment:
You probably want to use those signals also for the LC TAP? In that case they should probably go into a separate core file (or even somewhere totally else, probably best to do that once you know the final layout of the LC TAP).

Reply:
Agreed. Maybe prim directory with separate core file is a good candidate.